src/main/java/org/apache/maven/plugins/war/packaging/AbstractWarPackagingTask.java

```diff
@@ -477,6 +486,18 @@ protected String getArtifactFinalName( WarPackagingContext context, Artifact art
 
     }
 
+    /**
+     * Determine whether a file is a properties file or not.
+     *
+     * @param file The file to check
+     * @return <code>true</code> if the file is a properties file, otherwise <code>false</code>
+     * @since 3.4.0
+     */
+    private boolean isPropertiesFile( File file )
+    {
+        return file != null && file.isFile() && file.getName().endsWith( ".properties" );
```

Comment:
This now bacially duplicates `isXmlFile(File)`. Why refactor to a single method and pass two args to that new method?